usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -225,7 +238,13 @@ def test_cluster_unmanage_valid(
             Graphite contains no data related to health of tested cluster.
 
         """
-    for cluster in tendrl_api.get_cluster_list():
+    for i in range(15):
+        cluster_list = tendrl_api.get_cluster_list()
+        if len(cluster_list) > 0:
+            break
+        else:
+            time.sleep(10)
```

Comment:
I would add `else` statement for the loop `for i in range(15):` on line 241 and raise some exception there In other words, handle situation, when there is no cluster in cluster_list after 150s.

Reply:
This should be handled by the next pytest check

Reply:
Which one? There is directly another loop over the `cluster_list`, and if it is empty, `unmanaged_cluster` variable will not be set which will leads to the same error about `UnboundLocalError: local variable 'unmanaged_cluster' referenced before assignment`.

Reply:
Yeah, @fbalak we need to be able to report a case when the list is still empty after all the for loop waiting ...